packages/client/src/i18n/messages/views/performance.ts

```diff
@@ -326,6 +326,12 @@ const messagesToDefine = {
     defaultMessage: 'Births',
     description: 'Label for birth event option'
   },
+
```

Comment:
This can now be removed as we're not adding the `Event` dropdown, right?

Reply:
I think this is for @favour234 to remove.

Reply:
yes..would update.